config/benchmark_configs/amip_diagedmf.yml

```diff
@@ -2,6 +2,7 @@ FLOAT_TYPE: "Float32"
 aerosol_radiation: true
 atmos_config_file: "config/benchmark_configs/climaatmos_diagedmf.yml"
```

Comment:
This config (and `amip_diagedmf_io.yml`) take the atmosphere setup from `climaatmos_diagedmf.yml` and `climaatmos_diagedmf_io.yml` in this same directory, respectively. CO2 doesn't need to be specified here because it's already specified in the atmos-specific configs

Reply:
Thank you, I removed it from the two files.